include/barebox.mk

```diff
@@ -0,0 +1,102 @@
+PKG_NAME ?= barebox
```

Comment:
This file looks very similar to the u-boot file, would it be possible to extend the include/u-boot.mk file to work with both u-boot and barebox?

Reply:
Yes.